apertium-eng.eng.dix

```diff
@@ -26683,6 +26692,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="lower">           <i>lower</i><par n="accept__vblex"/></e>
 <e lm="lug">             <i>lug</i><par n="plug__vblex"/></e>
 <e lm="lunch">           <i>lunch</i><par n="approach__vblex"/></e>
+<e lm="lunge">           <i>lunge</i><par n="accept__vblex"/></e>
```

Comment:
"accept-ed" but "lunge-d", should be a different paradigm?

Reply:
I fixed it now